slidedeck/src/slides/04-react-router/00-react-router.pug

```diff
@@ -205,7 +203,7 @@ section
   p Where we were
   p.component.red this.props.history...
   p Where we are going
-  p.component.red &lt;Link to={location}/&gt;
+  p.component.red Link to={location}
```

Comment:
This looks a little odd to me without the <>

Reply:
Probs agreed.